modules/openapi-generator/src/main/resources/kotlin-spring/api_test.mustache

```diff
@@ -24,7 +25,7 @@ class {{classname}}Test {
     *          if the Api call fails
     */
     @Test
-    fun {{operationId}}Test() {
+    fun {{operationId}}Test() = runBlockingTest {
```

Comment:
on non reactive this test will not compile because the import runBlockingTest is not done.
maybe
`fun {{operationId}}Test() {{#reactive}}= runBlockingTest {{/reactive}}{`

Reply:
Done